api/src/main/java/io/opentelemetry/api/common/AttributesBuilder.java

```diff
@@ -155,4 +155,28 @@ public AttributesBuilder putAll(Attributes attributes) {
     attributes.forEach(this::put);
     return this;
   }
+
+  private static List<Double> toList(double... values) {
```

Comment:
Oh well. I wish Java had **real** generics :/

Reply:
I'm sure we'll get them with java 21.  😂